lighthouse-core/audits/metrics/first-contentful-paint-3g.js

```diff
@@ -60,7 +56,7 @@ class FirstContentfulPaint3G extends Audit {
         context.options.scoreMedian
       ),
       rawValue: metricResult.timing,
-      displayValue: str_(i18n.UIStrings.seconds, {timeInMs: metricResult.timing}),
+      displayValue: `${metricResult.timing}s`,
```

Comment:
Is there a way to statically fill in i18n formatted to use timeInMs with formatting?  I figured this might be enough since we aren't really i18n'ing it.